completion/utilities.py

```diff
@@ -0,0 +1,21 @@
+#Python imports
+from __future__ import unicode_literals, absolute_import
+
+#Project imports
+from .exceptions import UnavailableCompletionData
+from .models import BlockCompletion
+from .waffle import visual_progress_enabled
+
+#Library imports
+
+
+def get_key_to_last_completed_course_block(user, course_id):
+    if not visual_progress_enabled(course_id):
+        raise UnavailableCompletionData
+
+    last_completed_block = BlockCompletion.get_latest_block_completed(user, course_id)
+
+    if last_completed_block is None:
+        return last_completed_block.block_key
```

Comment:
How can something that is `None` have a `block_key` attribute?

Reply:
Good catch – this is a typo.